x-pack/plugins/endpoint/public/components/endpoints_page.tsx

```diff
@@ -148,13 +148,15 @@ export const EndpointsPage = () => {
             defaultFields={[`_source`]}
             updateOnChange={handleUserFilteredData}
           />
-          <EuiBasicTable
-            items={showFiltered ? filteredEndpoints : endpoints}
-            columns={columns}
-            pagination={paginationSetup}
-            sorting={sortingSetup}
-            onChange={handleTableChange}
-          />
+          <div data-test-subj="endpointsListTable">
```

Comment:
can this data prop be put on the component directly, instead of adding a wrapping div? does that still work?

Reply:
@pzl I added it directly to the search bar component. Let me try doing the same to the table.